cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -510,6 +512,7 @@ private void execute() {
         LOG.error("Executor got exception during execution", t);
       } finally {
         _loadMonitor.resumeMetricSampling(String.format("Resumed-By-Cruise-Control-After-Completed-Execution (Date: %s)", currentUtcDate()));
+        RESULT_LOG.info(String.format("Proposal execution:[%s] ended.", _uuid));
```

Comment:
The message `Proposal execution:[%s]` is not clear that it is printing UUID. Can we make it clear that this is `UUID`?